src/main/webapp/index-script.js

```diff
@@ -531,8 +521,59 @@ function voteOnReview(review) {
   const request = '/vote?comment_id=' + review.id + 
     '&up=' + review.positive + '&down=' + review.negative;
   fetch(request, {method:"POST"}).then((results) => {
-    console.log(results);
-      document.getElementById("up" + review.id).innerHTML = "&#128077;" + review.positive + " ";
-      document.getElementById("down" + review.id).innerHTML = "&#128078;" + review.negative + " ";
+    document.getElementById("up" + review.id).innerHTML = "&#128077;" + review.positive + " ";
+    document.getElementById("down" + review.id).innerHTML = "&#128078;" + review.negative + " ";
   });
+}
+
+/* Completes a thumbs up vote by either adding a new upvote or switching the user's current vote */
+function upvoteClick(review, currUser){
```

Comment:
No need to address this in this PR, but in general, having positive clauses first helps with readaiblity. e.g. This can be re-written as:


if (currUser == null). {
  return;   // Here we exist early to reduce nesting.
}
if (review.positiveVoters.includes(currUser) {
  // Do something
} else {
  // Do something
}

Reply:
Good to know - I will make sure to implement that / keep in mind in future PR's!